docs/package.json

```diff
@@ -27,7 +27,7 @@
     "gray-matter": "^4.0.3",
     "lodash": "^4.17.21",
     "mdx-prism": "^0.3.3",
-    "next": "12",
+    "next": "latest",
```

Comment:
Should we set this to `^12.0.0` so it doesn't (one day) bump up to v13 that Amplify Console wouldn't support?

Reply:
Updated to ^12.0.7 since the latest has the security fixes in it.